spec/contracts_spec.rb

```diff
@@ -85,9 +83,20 @@
     end
 
     context "when owner class does not include Contracts" do
+      let(:error) {
+        # NOTE Unable to support this user-friendly error for ruby
```

Comment:
I tried to approach the problem with `inherited` hook, but it failed, because if you have no method error in class definition body in 1.8.7, then class will not be defined at all, and hook will not fire.